component/olm.jsonnet

```diff
@@ -0,0 +1,32 @@
+// main template for cilium
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+// The hiera parameters for the component
+local params = inv.parameters.cilium;
+
+local olmDir = 'dependencies/cilium/olm/cilium/cilium-olm/cilium-olm-master/manifests/cilium.v%s/' % params.olm.version;
+
+local olmFiles = std.map(
+  function(name) {
+    filename: name,
+    contents: std.parseJson(kap.yaml_load(olmDir + name)),
```

Comment:
This currently makes the implicit assumption that each file contains exactly one Kubernetes manifest.

Reply:
Yes. I'd leave it like that since the compilation breaks if there's more than one manifest in a file. We'll see it in CI if we upgrade to a version with another structure.